deployment/suites/src/main/kotlin/org/codice/compliance/tests/suites/BasicTestsSuite.kt

```diff
@@ -13,12 +13,16 @@
  */
 package org.codice.compliance.tests.suites
 
-import org.codice.compliance.web.sso.PostLoginTest
-import org.codice.compliance.web.sso.RedirectLoginTest
+import org.codice.compliance.web.sso.PostSSOTest
+import org.codice.compliance.web.sso.RedirectSSOTest
+import org.codice.compliance.web.sso.error.PostSSOErrorTest
+import org.codice.compliance.web.sso.error.RedirectSSOErrorTest
 import org.junit.runner.RunWith
 import org.junit.runners.Suite
 
 @RunWith(Suite::class)
-@Suite.SuiteClasses(PostLoginTest::class,
-        RedirectLoginTest::class)
+@Suite.SuiteClasses(PostSSOTest::class,
```

Comment:
❓ Should we remove the negative tests from the test suite?

Reply:
Maybe we should have a positive and a negative suite?

Reply:
I would remove them from the suite, but once we fix relay state in DDF we're going to turn the negative relay state test back on right? There's another "negative" test that doesn't return a SAML error that we might turn on as well.

Reply:
We can separate them and still run the negative tests suite. At that point we will enable that test. We can also add a way to turn it off. Maybe by passing in a flag when we run Keycloak